karapace/schema_reader.py

```diff
@@ -247,26 +249,29 @@ def handle_messages(self) -> None:
                 try:
                     key = ujson.loads(msg.key.decode("utf8"))
                 except ValueError:
-                    self._log.exception("Invalid JSON in msg.key: %r, value: %r", msg.key, msg.value)
+                    self._log.exception("Invalid JSON in msg.key")
+                    self._log.log(TRACE, "Invalid JSON in msg.key: %r, value: %r", msg.key, msg.value)
                     continue
 
                 value = None
                 if msg.value:
                     try:
                         value = ujson.loads(msg.value.decode("utf8"))
                     except ValueError:
-                        self._log.exception("Invalid JSON in msg.value: %r, key: %r", msg.value, msg.key)
+                        self._log.exception("Invalid JSON in msg.value")
+                        self._log.log(TRACE, "Invalid JSON in msg.key: %r, value: %r", msg.key, msg.value)
                         continue
 
-                self._log.info("Read new record: key: %r, value: %r, offset: %r", key, value, msg.offset)
-                self.handle_msg(key, value)
-                self.offset = msg.offset
-                self._log.info(
-                    "Handled message, current offset: %r, ready: %r, add_offsets: %r",
```

Comment:
Has this message been intentionally removed? Maybe combine these logged values the the above logging statement.

Reply:
yes, this was intentionally removed, the previous line was the dispatch to the handlers, they extensive logging with additional context based on the message type